### ISSUE TITLE
Staging Review finding: File size limit hint text and error message do not match

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#125956]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Form Upload
**Product:** Form Upload
**Feature:** Multi-file upload (Adding supporting evidence)

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content with errors or inconsistencies.](https://design.va.gov/about/experience-standards/#user-encounters-accurate-consistent-content)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility), @erinrwhite (IA), @kkitagawa-bosch (content)

### Description

The file upload hint text indicates a file size limit of 25mb. When a user uploads a file greater than 25mb, the error text indicates that file sizes cannot be greater than 23.8mb. This mismatch may be confusing for users. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="539" height="465" alt="Hint text and error do not match" src="https://github.com/user-attachments/assets/a6c97441-106d-41f2-b9de-de4750275994" />

</details>

## Recommended action

Ensure that communicated information, user instructions, are consistent across every variation. In this case, the hint text and error message must be aligned to the actual limits of the form upload component. 

## References

- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG22/#info-and-relationships)
- **Modality:** N/A
- **Design System Component:** [Form - File input component](https://design.va.gov/components/form/file-input)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by tbaker1026** [comment]
<img width="471" height="318" alt="Image" src="https://github.com/user-attachments/assets/5932de33-b2ef-4fa7-96bb-94dc86aafe43" />
